src/hooks/useContract.ts

```diff
@@ -38,7 +38,8 @@ import useActiveWeb3React from 'hooks/useActiveWeb3React'
 import { useMemo } from 'react'
 import { NonfungiblePositionManager, Quoter, UniswapInterfaceMulticall } from 'types/v3'
 import { V3Migrator } from 'types/v3/V3Migrator'
-import { getContract } from 'utils'
+
+import { getContract } from '../utils'
```

Comment:
This is changed as an alternative to adding `utils` to paths for `tsconfig.lib.ts`.